repos/system_upgrade/common/actors/cephvolumescan/libraries/cephvolumescan.py

```diff
@@ -8,7 +8,7 @@
 from leapp.models import InstalledRPM
 
 CEPH_CONF = "/etc/ceph/ceph.conf"
-CONTAINER = "ceph-osd"
+CONTAINER = "ceph-.*osd"
```

Comment:
This looks a bit greedy to me, I would rather use:
```suggestion

CONTAINER = r"ceph-(\S*-)?osd-"
```